.github/workflows/package.yml

```diff
@@ -1,27 +1,35 @@
 on:
-  push:
-    branches:
-      - master
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
```

Comment:
I'm confused ! 😅 

Do we want everytime a PR is updadeted by dependabot to push a commit upstream? or should it be strictly after a PR is merged